lib/pipefy_message/broker/aws/configuration.rb

```diff
@@ -11,13 +11,27 @@ class ProviderConfig
         include Singleton
 
         def setup_connection
-          Aws.config.update(
-            endpoint: ENV["AWS_ENDPOINT"],
-            access_key_id: ENV["AWS_ACCESS_KEY_ID"],
-            secret_access_key: ENV["AWS_SECRET_ACCESS_KEY"],
-            region: ENV["AWS_REGION"] || "us-east-1",
-            stub_responses: ENV["AWS_CLI_STUB_RESPONSE"] || false
-          )
+          Aws.config.update(retrieve_config)
+        end
+
+        private
+
+        def aws_client_config?
+          ENV["ENABLE_AWS_CLIENT_CONFIG"] == "true"
```

Comment:
Is it's ok to use true how string here?

Reply:
I think so, if you take look at pipefy-core Kustomize env vars file (k8s/manifests/app-core-envs/app-core-configmap.yaml) we'll see that is already used in this way, so I just follow the current pattern. 

Do you see any problem on this approach?

Reply:
No, it's ok 😀